src/main/java/lotto/domain/WinningInfo.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import lotto.exception.DuplicatedBonusLottoNumberException;
+
+public class WinningInfo {
+
+  private final LottoNumbers winningLottoNumbers;
```

Comment:
```suggestion
  private final Lotto winningLotto;
```
여기도 이렇게 표현해 줄 수 있도록 만들어주면 더 좋을 것 같아요!

Reply:
위 코멘트와 같은 생각으로... ㅎㅎ Lotto대신 LottoNumbers를 넣어보았습니다.   Lotto를 사용하는게  더 나을까요 ??